styles/components/_autocomplete.scss

```diff
@@ -43,3 +43,8 @@
 .sprk-c-Autocomplete__result--active {
   background-color: $sprk-autocomplete-result-active-background-color;
 }
+
+/// Hide the Autocomplete Results
+.sprk-c-Autocomplete__results--hidden {
+  display: $sprk-autocomplete-results-hidden-display;
+}
```

Comment:
New class so we don't need to use `sprk-u-Display--none`

Reply:
@bobwise  thanks for adding the new class! Lets update it to match the `is-hidden` naming convention though!